src/main/java/frc/robot/Constants.java

```diff
@@ -33,12 +38,57 @@ public final class ArmConstants {
     }
 
     public static final class DriveConstants {
+
+        public static final double trackWidth = Units.inchesToMeters(17.75);
+        public static final double wheelBase = Units.inchesToMeters(23.75);
```

Comment:
have these numbers been actually measured? If not, let's make a comment that we need to measure these and that they are placeholder numbers